opendevin/runtime/docker/ssh_box.py

```diff
@@ -227,6 +231,23 @@ def __init__(
 
         self.timeout = timeout
         self.container_image = container_image or config.sandbox_container_image
+
+        self.use_agnostic_sandbox = "ghcr.io/opendevin/sandbox" not in self.container_image
+
+        if self.use_agnostic_sandbox:
+            images = self.docker_client.images.list()
+
+            image_exists = False
+            for image in images:
+                if AGNOSTIC_SANDBOX_IMAGE_NAME in image.tags:
```

Comment:
Maybe we can create name for the built sandbox using `self.container_image`?
Something like `agnostic_sandbox_image_name = self.contain_image + '_od'`?